allan_variance.py

```diff
@@ -120,3 +129,23 @@ def params_from_avar(tau, avar):
     prediction = A.dot(x) * avar
 
     return np.sqrt(x), prediction
+
+
+def allan_deviation(avar):
```

Comment:
According to DRY just link to ``allan_variance`` in ``allan_deviation`` docstring :)